azure/functions/servicebus.py

```diff
@@ -102,6 +120,26 @@ def to(self) -> typing.Optional[str]:
     def user_properties(self) -> typing.Dict[str, object]:
         return self.__user_properties
 
+    @property
+    def metadata(self) -> str:
```

Comment:
Yeah, this is a good catch, because I was thinking json is more intuitive for customer to view all data at once. But now I'm changing it to Python dictionary for ease of access.